bin/build

```diff
@@ -57,10 +60,11 @@ pushd $IMAGE_DIR
     echo "Building $image: $cmd"
     $cmd
   done
-
+ 
```

Comment:
Can you remove the trailing whitespace at the end of this line?

Reply:
done